www/apps/docs/content/user-guide/tips/languages.md

```diff
@@ -23,6 +23,7 @@ Languages listed in this section are contributed by the Medusa community. So, th
 :::
 
 - Bosanski
+- Čeština
```

Comment:
rm: This needs to be added in a separate PR, otherwise it gets published to our docs as soon as the PR is merged

Reply:
Could you mark those changes as resolved? It seems [a check is waiting for these changes to be confirmed](https://github.com/medusajs/medusa/pull/5822/checks?check_run_id=19751722388)

Reply:
@kasperkristensen Could you mark the changes as resolved? It seems [a check is waiting for them to be resolved](https://github.com/medusajs/medusa/pull/5822/checks?check_run_id=19751722388)